HomeWork-1/task-1.js

```diff
@@ -0,0 +1,17 @@
+const loop = function(times = 0, callback = null) {
+
+    if(arguments[1] != null){
```

Comment:
```suggestion
    if(typeof arguments[1] !== 'function'){
```
можно еще так реализовать:)

Reply:
хорошо